actors/inferenceapi/src/lib.rs

```diff
@@ -304,8 +304,6 @@ async fn predict(
     link_name: &str,
     tensor: Tensor,
 ) -> RpcResult<InferenceOutput> {
-    debug!("Deserialized input tensor: {:?}", tensor);
```

Comment:
This was outputting so much text to the terminal it would clear the entire terminal's writeback. And apparently we don't have trace level logging in our interface. We could now though! That was a limitation of elixir log levels